core/src/main/java/jenkins/management/NodesLink.java

```diff
@@ -49,6 +53,12 @@ public String getDescription() {
         return Messages.NodesLink_Description();
     }
 
+    @CheckForNull
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
+    }
+
```

Comment:
No special permissions are needed to access this anyway: https://ci.jenkins.io/computer/

Reply:
✔️ still accessible to user with just Overall/Read

Reply:
So an improvement then?